xds/src/main/java/com/linecorp/centraldogma/xds/internal/ControlPlanePluginConfig.java

```diff
@@ -26,7 +28,7 @@ public final class ControlPlanePluginConfig extends AbstractPluginConfig {
     /**
      * Creates a new instance.
      */
-    public ControlPlanePluginConfig(@Nullable Boolean enabled) {
+    public ControlPlanePluginConfig(@JsonProperty("enabled") @Nullable Boolean enabled) {
```

Comment:
Don't we need `@JsonCreator`?

Reply:
It's not needed when there's only one constructor. Let me add it though for clarity.